### PR TITLE
[hi] Localize content/en/docs/reference/glossary/uid.md in hindi

### PR DESCRIPTION
**Pull request by KashishLakhara04**
fixes: #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/uid.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: यूआईडी (UID)
+id: uid
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/names
+short_description: >
+  विशिष्ट वस्तुओं की पहचान करने के लिए एक कुबेरनेट्स सिस्टम-जनित स्ट्रिंग।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/uid.md:7]:
```suggestion
  विशिष्ट ऑब्जेक्ट्स की पहचान करने के लिए एक कुबेरनेट्स सिस्टम-जनित स्ट्रिंग।
```

File: content/hi/docs/reference/glossary/uid.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+aka: 
+tags:
+- fundamental
+---
+ विशिष्ट वस्तुओं की पहचान करने के लिए एक कुबेरनेट्स सिस्टम-जनित स्ट्रिंग।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/uid.md:12]:
```suggestion
 विशिष्ट ऑब्जेक्ट्स की पहचान करने के लिए एक कुबेरनेट्स सिस्टम-जनित स्ट्रिंग।
```

File: content/hi/docs/reference/glossary/uid.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<!--more--> 
+
+एक कुबेरनेट्स क्लस्टर के पूरे जीवनकाल में बनाई गई प्रत्येक वस्तु का एक अलग UID है। इसका उद्देश्य समान संस्थाओं की ऐतिहासिक घटनाओं के बीच अंतर करना है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/uid.md:15]:
```suggestion
एक कुबेरनेट्स क्लस्टर के पूरे जीवनकाल में बनाई गई प्रत्येक ऑब्जेक्ट का एक अलग UID होता है। इसका उद्देश्य समान इकाइयों की ऐतिहासिक घटनाओं के बीच अंतर करना है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/uid.md (hunk 3 above)
Comment (KashishLakhara04) [reply content/hi/docs/reference/glossary/uid.md]: @anubha-v-ardhan  All the suggested changes have been applied now.

**Review by Babapool — COMMENTED** [review]
Thanks!
/lgtm

**Pull request merged**